core/templates/services/platform-feature.service.spec.ts

```diff
@@ -208,6 +208,70 @@ describe('PlatformFeatureService', () => {
     }));
   });
 
+  describe('clearSavedResults', () => {
+    it('should remove sessionStorage item if nativeWindow exists', () => {
+      platformFeatureService = TestBed.inject(PlatformFeatureService);
+
+      const removeItemSpy = spyOn(
+        windowRef.nativeWindow.sessionStorage, 'removeItem'
+      );
+
+      mockSessionStore({
+        // This throws "TS2341". We need to suppress this error because
+        // The 'SESSION_STORAGE_KEY' member is private
```

Comment:
Here and below (multiple places), change "The" to "the" because it continues on from the previous line.

Reply:
@seanlip, sorry, my bad. I'll keep that in mind. PTAL.